lib/ao_host.js

```diff
@@ -222,8 +238,12 @@ class AOHost extends AsyncEventEmitter {
   }
 
   onMetaReload () {
-    this.cleanState()
-    this.emit('meta:reload')
+    const sinceLastMessage = Date.now() - this.lastMessageTime
+    // check if packet delay timed out
+    if (sinceLastMessage > this.packetWDDelay) {
```

Comment:
Won't this result in algo orders being stopped and users not being given the option to resume/cancel them in certain cases?

Reply:
Fixed. Added packetWDDelay as ws timeout for dms control.